### PR TITLE
test work!

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

File: tests/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "todomvc-tests",
+  "private": true,
```
Comment (arthurvr) [inline tests/package.json]: There's no deal with including `name` when you make something private.

Comment (samccone) [inline tests/package.json]: :)

**Comment by arthurvr** [comment]
It does seem strange that we're going to have a `tests/` directory and that the actual tests aren't there but in `browser-tests/`.

**Comment by arthurvr** [comment]
I don't really like the `util/` directory name as well. It gives the impression that actual code lives in there. That's not really true, most people will never need these scripts.

**Comment by sindresorhus** [comment]
Maybe:

`browser-tests` => `test`
`util` => `tooling`
`tests` => `tooling`

**Comment by arthurvr** [comment]
Yes. I wonder if we can dump that `travis-runner.sh` in `tooling` then as well?

**Comment by samccone** [comment]
yeah i can drop the util folder and just make a func since it is only used in one spot

**Comment by samccone** [comment]
travis-runner is actually a bad name now.. and can just be test-runner.sh since it will run everything :)

File: test-runner.sh [hunk 2]
```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 set -o pipefail
 
+
+get_changes ()
+{
+  git remote add current https://github.com/tastejs/todomvc.git && \
```
Comment (arthurvr) [inline test-runner.sh]: why does this file use 2 spaces indentation?

**Comment by samccone** [comment]
all updated @arthurvr

**Comment by arthurvr** [comment]
Yiippeeee!

**Pull request merged**